core/src/test/java/org/infinispan/stats/SingleStatsTest.java

```diff
@@ -64,10 +69,10 @@ protected void configure(ConfigurationBuilder cfg) {
             // Binary key/value size is 128 bytes
             size *= 128;
          } else {
-            // Off heap key/value size is 64 bytes
-            size = UnpooledOffHeapMemoryAllocator.estimateSizeOverhead(size * 64);
+            // Off heap key/value size is 80 bytes
```

Comment:
So there's an extra 80-64=16 bytes even when the `MetaParamsInternalMetadata` isn't used?